modules/data/address_fill.components.json

```diff
@@ -9,5 +9,19 @@
         "selectorData": "input[type=\"{name}\"]",
         "strategy": "css",
         "groups": []
+    },
+
+    "context": "chrome",
```

Comment:
This `"context": "chrome"` should only be at the top of the file once. You should remove the ones that are redudant

Reply:
Will do that, thanks.